include/CMakeLists.txt

```diff
@@ -101,9 +100,7 @@ install( TARGETS backend_headers_nodefs EXPORT GraphBLASTargets
 	INCLUDES DESTINATION "${INCLUDE_INSTALL_DIR}"
 )
 
-install( TARGETS alp_utils_headers EXPORT GraphBLASTargets
-	INCLUDES DESTINATION "${INCLUDE_INSTALL_DIR}"
-)
+install( TARGETS alp_utils_headers EXPORT GraphBLASTargets )
```

Comment:
Why's `INCLUDES DESTINATION` removed here but added for other targets later on (e.g., transition_headers?)

Reply:
This has changed in the last commit, let me explain the motivation behind both changes.

This line was removed because it is redundant: the utils library has the same include paths as ALP, already installed at line 100 of the same file.

The second commit adds it again because the utils library is ideally a separate library, so this way we have independently installed targets: a user could ideally link utils without linking ALP, or vice versa. A comment explains this.

Reply:
Got it, many thanks!